controller/volume_controller.go

```diff
@@ -1940,7 +1943,8 @@ func (vc *VolumeController) createCronJob(v *longhorn.Volume, job *types.Recurri
 								},
 							},
 							ServiceAccountName: vc.ServiceAccount,
-							RestartPolicy:      v1.RestartPolicyOnFailure,
+							//RestartPolicy:      v1.RestartPolicyOnFailure,
+							RestartPolicy: v1.RestartPolicyNever,
```

Comment:
?

Reply:
My bad. I forgot to remove debugging code